src/components/Nav/Nav.module.css

```
@@ -0,0 +1,98 @@
+/* Copyright 2023 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.nav-bar {
+  border-block-end: var(--cpd-border-width-1) solid var(--cpd-color-gray-400);
+  margin: var(--cpd-space-6x) 0;
+  padding: 0;
+}
+
+.nav-bar-items {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  gap: var(--cpd-space-3x);
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav-tab {
+  padding: var(--cpd-space-4x) 0;
+  position: relative;
+}
+
+/* Underline effect */
+.nav-tab::before {
+  content: "";
+  position: absolute;
+  inset-block-end: 0;
+  inset-inline: 0;
+  block-size: 0;
+  border-radius: var(--cpd-radius-pill-effect) var(--cpd-radius-pill-effect) 0 0;
+  background-color: var(--cpd-color-bg-action-primary-rest);
+  transition: height 0.1s ease-in-out;
+}
+
+.nav-tab[data-current]::before {
+  /* This is not exactly right: designs says 3px, but there are no variables for that */
+  block-size: var(--cpd-border-width-4);
+}
+
+.nav-item {
+  padding-block: var(--cpd-space-1x);
+  padding-inline: var(--cpd-space-2x);
+  border-radius: var(--cpd-radius-pill-effect);
+  cursor: pointer;
+  appearance: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: var(--cpd-space-2x);
+  box-sizing: border-box;
+  background: transparent;
+  border: 0;
+  font: var(--cpd-font-body-md-regular);
+  color: var(--cpd-color-text-secondary);
+  text-decoration: none;
+}
+
+@media (hover) {
+  .nav-item:not([disabled]):hover {
+    color: var(--cpd-color-text-primary);
+    background-color: var(--cpd-color-bg-subtle-secondary);
+  }
+}
+
+.nav-item:focus {
```

```suggestion
.nav-item:focus-visible {
```

So that the focus ring doesn't appear for pointing device users